packages/destination-actions/src/destinations/saleswings/index.ts

```diff
@@ -0,0 +1,35 @@
+import type { DestinationDefinition } from '@segment/actions-core'
+import { apiBaseUrl } from './api'
+import type { Settings } from './generated-types'
+
+import submitEvent from './submitEvent'
+
+const destination: DestinationDefinition<Settings> = {
+  name: 'Saleswings (Actions)',
+  slug: 'actions-saleswings',
+  mode: 'cloud',
+
+  authentication: {
+    scheme: 'custom',
+    fields: {
+      apiKey: {
+        label: 'API Key',
+        description: 'Segment.io API key for your SalesWings project',
```

Comment:
Descriptions should end with a `.`.
```suggestion
        description: 'Segment.io API key for your SalesWings project.',
```